domain/vod/src/main/java/model/Language.kt

```diff
@@ -0,0 +1,11 @@
+package model
+
+enum class Language(val code: String) {
+    ENGLISH("en"),
+    ARABIC("ar");
+
+    companion object {
+        fun fromCode(code: String): Language? =
+            Language.entries.find { it.code == code }
+    }
```

Comment:
There's no need for this if you need to map data to a domain, you can create a mapper